install/kubernetes/helm/istio/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
```

Comment:
I think you do not need to create roles and rolebindings, but need to update existing clusterrole and clusterrolebindings for all deployments in istio helm chart.

Reply:
I see, this looks good, and the rolebindings here can make sure all services account under `istio-system` (default namespace) have the permission.

Please also append some test result with Kube 1.11 and we can merge it then.

Reply:
@gyliu513 Test on a kubernetes 1.11 cluster, find that the fix can't be always effective,because the helm-tiller won't guarantee the creating order of psp, as a result, some pods can be created and run normally(because they are created after psp), some are not(with errorCreatingContainerConfigError).
If I manually delete all failed pods, they will be recreated successfully.

Another option is to use pre-install hook create the psp and post-delete hook to delete the psp.